core/src/main/scala/org/bitcoins/core/protocol/ln/LnTags.scala

```diff
@@ -75,36 +75,33 @@ object LnTag {
       val u8 = UInt8.zero
     }
 
+    private def witnessFromU8(bytes: ByteVector, np: NetworkParameters) = {
+      val witSPK = bytes.size match {
```

Comment:
.length for intellij

Reply:
from `ByteVector.scala` implementation: 
```scala
  /**
   * Alias for [[size]].
   * @group collection
   */
  final def length: Long = size
```

So here it doesn't matter.